src/frethandfinger.ts

```diff
@@ -117,27 +115,26 @@ export class FretHandFinger extends Modifier {
       .map((fingering: Modifier, index: number) => note.addModifier(fingering, index));
   }
 
-  protected finger: string;
   protected xOffset: number;
   protected yOffset: number;
 
   constructor(finger: string) {
     super();
 
-    this.finger = finger;
-    this.width = 7;
+    this.setFretHandFinger(finger);
     this.position = Modifier.Position.LEFT; // Default position above stem or note head
     this.xOffset = 0; // Horizontal offset from default
     this.yOffset = 0; // Vertical offset from default
   }
 
   setFretHandFinger(finger: string): this {
-    this.finger = finger;
+    this.text = finger;
+    this.measureText();
```

Comment:
I still find it a bit confusing that we need to remember to call this.measureText() whenever we change the this.text.

Is it a big performance hit if we automatically update the measurements? I guess maybe there are situations where we want to keep updating the text and batch up all the changes before we call this.measureText() a single time?

Reply:
We definetively have to improve on that. #38 will help identifying the performace drivers.